### DISCUSSION TITLE
Searchable Select field do not emit global change event

### DISCUSSION THREAD

**Discussion by hybrid1969** [laravel/nova-issues#6523] [Q&A] [answered]
- Laravel Version: 11
- Nova Version: 4
- PHP Version: 8.2
- Database Driver & Version: n/a
- Operating System and Version: n/a
- Browser type and version: n/a

### Description:
Was this ever addressed -
Searchable Select field do not emit global change event #3569
https://github.com/laravel/nova-issues/issues/3569

@dalmarcolucas did you ever find a workaround as I tried your suggestions without success and this still seems to still be an issue, which is a major miss in my opinion?

**Comment by crynobone** [comment, marked as answer]
Hi there,

Field events are now strictly for internal usage and dependent fields. You can try to listen to it, but the API is locked for internal use only. With the introduction of inline relationship form and repeater, it becomes unreliable to provide global events as field name will not always be unique. This wasn't an issue in Nova 3 as described in #3569 since such features is not available at the time.

**Reply by hybrid1969** [reply]
Appreciate the explanantion and will look for either a workaround or making my own custom field
